schemas/context/weblink.schema.json

```diff
@@ -27,12 +28,13 @@
         "xdm:URL": {
           "title": "URL",
           "type": "string",
-          "description": "The link URL."
+          "description": "The actual link/URL used for this web interaction"
```

Comment:
This is adding further confusion, because while a link *points to* a URL, and *exists at* another URL, it *is not* a URL itself. If we want to model this, we need to make clear which URL this is:
- the one where the interaction is recorded
- the one where the interaction was triggered
- the one that was being linked to

Reply:
Its primary usage is "the one where the interaction was triggered". It is actually recorded after the link is fully executed if memory serves.